cluster/ubuntu/util.sh

```diff
@@ -508,6 +505,12 @@ function provision-masterandnode() {
       cp ~/kube/minion/* /opt/bin/
 
       service etcd start
+      service kube-apiserver start
```

Comment:
You won't suffer. The script takes care of this for you.
Ref https://github.com/kubernetes/kubernetes/blob/master/cluster/ubuntu/master/init_conf/kube-apiserver.conf#L7